src/devtoolsPanel.ts

```diff
@@ -514,7 +508,7 @@ export class DevToolsPanel {
                 ">
             </head>
             <body>
-                <iframe id="devtools-frame" frameBorder="0" src="${cdnBaseUri}?experiments=true&theme=${theme}&headless=${enableScreencast}&standaloneScreencast=${standaloneScreencast}&cssMirrorContent=${cssMirrorContent}"></iframe>
+                <iframe id="devtools-frame" frameBorder="0" src="${cdnBaseUri}?experiments=true&theme=${theme}&standaloneScreencast=true&cssMirrorContent=${cssMirrorContent}"></iframe>
```

Comment:
We should stage a patch to deprecate the `standaloneScreencast` queryparam in dtfe

Reply:
@hkal don't we need to keep that param for the DevTools Visual Studio extension (not VS Code)? This extension continues to opt-in so other hosts that don't have their own standalone screencast can use the one built-in with DevTools.

Reply:
Oh right - totally slipped my mind. No followup is needed then.